lib/resource-mapper.js

```diff
@@ -84,28 +84,29 @@ class ResourceMapper {
 
   // Maps the request for a given resource and representation format to a server file
   // Will look for an index file if a folder is given and searchIndex is true
-  async mapUrlToFile ({ url, contentType, createIfNotExists, searchIndex = true }) {
+  async mapUrlToFile ({ url, contentType, createIfNotExists, searchIndex }) {
     // Parse the URL and find the base file path
     const { pathname, hostname } = this._parseUrl(url)
     const filePath = this.resolveFilePath(hostname, decodeURIComponent(pathname))
     if (filePath.indexOf('/..') >= 0) {
       throw new Error('Disallowed /.. segment in URL')
     }
-    let isIndex = searchIndex && filePath.endsWith('/')
+    let isFolder = filePath.endsWith('/')
```

Comment:
Is this actually an assumption we can make? Or should it be passed in explicitly as a parameter?

Reply:
`const`?